### PR TITLE
Adding Docker Capabilities

### PR DESCRIPTION
**Pull request by ChuckMac**
Add docker configuration so this can be run in a docker-compose schema.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+ENV_VARS_FILE="/home/app/.cron_env"
+
+echo "Dumping env variables into ${ENV_VARS_FILE}"
+eval $(printenv | awk -F= '{print "export " "\""$1"\"""=""\""$2"\"" }' >> ${ENV_VARS_FILE})
+chmod +x ${ENV_VARS_FILE}
```
Comment (iloveitaly) [inline entrypoint.sh:7]: Curious, why do we need to dump vars here?

**Comment by iloveitaly** [comment]
This is an awesome change! Thanks for the contribution.

**Comment by ChuckMac** [comment]
Additional updates:

- Remove extra env dump code
- Don't build the image twice
- Move time wait to env variable instead of hardcode

**Comment by iloveitaly** [comment]
@ChuckMac awesome! Merging this in. Would be great to add a changelog entry for this if you could in a separate PR.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM node:12-buster
+
+## Chrome
+RUN apt-get update && apt-get install -y gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
```
Comment (iloveitaly) [inline Dockerfile:4]: @ChuckMac curious: which environment is this Dockerfile for? I'm building one for the raspberry Pi, and it's looking quite a bit different.

**Comment by ChuckMac** [comment]
Hi @iloveitaly not quite sure I understand the question…. the image being built from is the node12 Debian Linux (Buster release ).  However Docker itself is environment independent and it should be able to run on a raspberry pi.

**Pull request merged**